### PR TITLE
Uninstall package function added.

### PR DESCRIPTION
**Pull request by stephanmg**
This may close issue #7 after review.

### CODE REVIEW

File: ughub.py [hunk 1]
```diff
@@ -652,6 +666,33 @@ def GetCurrentRemoteGitURLs(pkg, origin="origin"):
...
+        print("Please specify a package name. See 'ughub help uninstall'.")
+        return
+
+    all_packages = LoadPackageDescs()
+    for package in packageNames:
+        for p in all_packages:
+            if p["name"] == package:
+                if PackageIsInstalled(p):
+                     import shutil
+                     shutil.rmtree(os.path.join(GetRootDirectory(), p["name"]))
```
Comment (sreiter) [inline ughub.py:694]: I think this won't work since, e.g., 'plugin' or 'apps' paths are not considered.

**Comment by sreiter** [comment]
Thanks for your proposal. Please have a look at my comment on issue #7.

Furthermore, the pull request contains a lot more changes than just the additional functions required to implement the 'uninstall' command. I would really appreciate if you could create a merge request which considers #7 and only contains necessary changes/additions.

Thanks and kind regards,
Sebastian

**Comment by stephanmg** [comment]
Okay @sreiter. Depending on how you decide on the issue #9 I'll make a new pull request and close this pull request. Just leave me a small note here if you decide to convert to spaces in favor of tabs, see discussion #9 too.

**Comment by stephanmg** [comment]
I will close this pull request and open a new with only the installPackage function changes.

**Pull request closed without merging**